backend/src/main/java/com/capstone/moneytree/service/impl/DefaultUserService.java

```diff
@@ -154,6 +155,10 @@ public User updateUser(User userToUpdate, User user) {
             userToUpdate.setAvatarURL(user.getAvatarURL());
         }
 
+        if (user.getCoverPhotoURL() != null) {
+            userToUpdate.setCoverPhotoURL(user.getCoverPhotoURL());
```

Comment:
This was added and needs a test for this path. Need to keep the coverage high :P